data/src/main/java/org/apache/iceberg/data/avro/DataReader.java

```diff
@@ -58,8 +58,8 @@ private DataReader(Schema readSchema) {
   }
 
   @Override
-  public void setSchema(Schema fileSchema) {
-    this.fileSchema = Schema.applyAliases(fileSchema, readSchema);
+  public void setSchema(Schema schema) {
```

Comment:
I believe in previous PRs @mccheah frequently used `fileSchema` -> `newFileSchema` type of renames to avoid hiding fields in builders. Would it make sense to make it consistent?

Reply:
Will update